### PR TITLE
Return properly encoded value by eth_getStorageAt RPC method for non-existing keys

### PR DESCRIPTION
**Pull request by casiojapi**
## Description
Updated eth_getStorageAt to return a 32-byte zero value for non-existing keys, aligning with geth client behavior.

## Motivation and Context
Ensures compatibility with Ethers.js and other clients like Geth, which expect a 32-byte zero value for non-existing keys.

## How Has This Been Tested?
Tested in local development environment; unit tests updated to reflect new return value.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

File: rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java [hunk 1]
```diff
@@ -487,7 +487,7 @@ private String eth_getStorageAt(HexAddressParam address, HexNumberParam storageI
                     .getStorageValue(addr, DataWord.valueOf(HexUtils.strHexOrStrNumberToByteArray(storageIdx.getHexNumber())));
 
             if (sv == null) {
-                s = "0x0";
+                s = "0x" + String.join("", Collections.nCopies(64, "0"));
```
Comment (Vovchyk) [inline rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java:490]: just an idea, could it simply be in the form of the string literal (probably defined as a constant) - same as in the tests, like this? - `0x0000000000000000000000000000000000000000000000000000000000000000`

Comment (casiojapi) [reply rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java:490]: Yeah, maybe that makes it more readable. I'll update it.

**Comment by casiojapi** [comment]
pipeline:run

**Comment by casiojapi** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
A small suggestion, for the test files you could create a constant like you did for [Web3Impl](https://github.com/rsksmart/rskj/pull/2205/files#diff-6559cff582e5f737e87f69c2dcae4591c06166f215784b542b23747e13c91cc4R89) so you don't have to write the string literal more than once, you could also make the constant in Web3Impl public and use that one. Aside form that nit, it looks good to me.

**Review by Vovchyk — APPROVED** [review]

**Review by jeremy-then — COMMENTED** [review]
Left a couple of comments.

By the way, why is there no documentation for `rsk_` methods in `rskj/doc/rpc/methods`?

**Pull request closed without merging**